### DISCUSSION TITLE
Thresholding to define positive and negative cells for markers

### DISCUSSION THREAD

**Discussion by akk01** [BodenmillerGroup/steinbock#209] [General]
Steinbock generates expression data (signal intensities in csv file), which represents a spectrum of individual marker expression levels across cells. For cell phenotyping,  often we look at more than one markers and determine single positive, double positive, positive and negative etc. Does steinbock provide a function to define threshold of individual markers. Please advise me.

**Comment by nilseling** [comment]
Hi @akk01 

`steinbock` does not support this task as the toolbox was mainly written for image processing rather than downstream analysis. You can have a look at https://bodenmillergroup.github.io/IMCDataAnalysis/cell-phenotyping.html to see if there's anything that can help you in terms of cell phenotyping.
